### ISSUE TITLE
Build breaks with clang-19: no member named 'coord_type' in 'PointTopology<T, N>'; did you mean 'coords_type'?

### ISSUE THREAD

**Issue by yurivict** [Alpine-DAV/ascent#1422]
```
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/runtimes/expressions/ascent_blueprint_topologies.cpp:163:12: error: no member named 'coord_type' in 'PointTopology<T, N>'; did you mean 'coords_type'?
  163 |   if(this->coord_type == "uniform")
      |            ^~~~~~~~~~
      |            coords_type
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/runtimes/expressions/ascent_blueprint_topologies.hpp:56:21: note: 'coords_type' declared here
   56 |   const std::string coords_type;
      |                     ^
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/runtimes/expressions/ascent_blueprint_topologies.cpp:180:17: error: no member named 'coord_type' in 'PointTopology<T, N>'; did you mean 'coords_type'?
  180 |   else if(this->coord_type == "rectilinear")
      |                 ^~~~~~~~~~
      |                 coords_type
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/runtimes/expressions/ascent_blueprint_topologies.hpp:56:21: note: 'coords_type' declared here
   56 |   const std::string coords_type;
      |                     ^
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/runtimes/expressions/ascent_blueprint_topologies.cpp:192:17: error: no member named 'coord_type' in 'PointTopology<T, N>'; did you mean 'coords_type'?
  192 |   else if(this->coord_type == "explicit")
      |                 ^~~~~~~~~~
      |                 coords_type
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/runtimes/expressions/ascent_blueprint_topologies.hpp:56:21: note: 'coords_type' declared here
   56 |   const std::string coords_type;
      |                     ^
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/runtimes/expressions/ascent_blueprint_topologies.cpp:206:27: error: no member named 'coord_type' in 'PointTopology<T, N>'; did you mean 'coords_type'?
  206 |                  << this->coord_type << "' for point topology '" << topo_name
      |                           ^~~~~~~~~~
      |                           coords_type
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/utils/ascent_logging.hpp:62:25: note: expanded from macro 'ASCENT_ERROR'
   62 |     ascent_oss_error << msg;                                        \
      |                         ^
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/runtimes/expressions/ascent_blueprint_topologies.hpp:56:21: note: 'coords_type' declared here
   56 |   const std::string coords_type;
      |                     ^
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/runtimes/expressions/ascent_blueprint_topologies.cpp:216:12: error: no member named 'coord_type' in 'PointTopology<T, N>'; did you mean 'coords_type'?
  216 |   if(this->coord_type == "uniform")
      |            ^~~~~~~~~~
      |            coords_type
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/runtimes/expressions/ascent_blueprint_topologies.hpp:56:21: note: 'coords_type' declared here
   56 |   const std::string coords_type;
      |                     ^
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/runtimes/expressions/ascent_blueprint_topologies.cpp:224:17: error: no member named 'coord_type' in 'PointTopology<T, N>'; did you mean 'coords_type'?
  224 |   else if(this->coord_type == "rectilinear")
      |                 ^~~~~~~~~~
      |                 coords_type
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/runtimes/expressions/ascent_blueprint_topologies.hpp:56:21: note: 'coords_type' declared here
   56 |   const std::string coords_type;
      |                     ^
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/runtimes/expressions/ascent_blueprint_topologies.cpp:237:17: error: no member named 'coord_type' in 'PointTopology<T, N>'; did you mean 'coords_type'?
  237 |   else if(this->coord_type == "explicit")
      |                 ^~~~~~~~~~
      |                 coords_type
/wrkdirs/usr/ports/science/ascent/work/ascent-0.9.3/src/libs/ascent/runtimes/expressions/ascent_blueprint_topologies.hpp:56:21: note: 'coords_type' declared here
   56 |   const std::string coords_type;
      |                     ^
```

[log](https://pkg-status.freebsd.org/beefy18/data/main-amd64-default/pea7acb655568_s65a4daeaf32/logs/ascent-visualizer-0.9.3.log)

Version: 0.9.3
FreeBSD 14.1

**Comment by cyrush** [comment]
resolved in #1408

**Comment by cyrush** [comment]
@yurivict thanks for the report, this should be resolved in ascent@develop and the next release.
